sdk/schemaregistry/azure-schemaregistry-avroserializer/CHANGELOG.md

```diff
@@ -1,19 +1,17 @@
 # Release History
 
-## 1.0.0b4 (Unreleased)
+## 1.0.0b4 (2021-11-11)
 
 ### Features Added
 
 - Async version of `AvroSerializer` has been added under `azure.schemaregistry.serializer.avroserializer.aio`.
-- `SchemaParseError`, `SchemaSerializationError`, and `SchemaDeserializationError` have been introduced under `azure.schemaregistry.serializer.avroserializer.exceptions` and will be raised for corresponding operations.
-  - `SchemaParseError` and `SchemaSerializationError` may be raised for errors when calling `serialize` on `AvroSerializer`.
-  - `SchemaParseError` and `SchemaDeserializationError` may be raised for errors when calling `deserialize` on `AvroSerializer`.
 
 ### Breaking Changes
 
-### Bugs Fixed
-
-### Other Changes
+- `SchemaParseError`, `SchemaSerializationError`, and `SchemaDeserializationError` have been introduced under `azure.schemaregistry.serializer.avroserializer.exceptions` and will be raised for corresponding operations.
+  - `SchemaParseError` and `SchemaSerializationError` may be raised for errors when calling `serialize` on `AvroSerializer`.
+  - `SchemaParseError` and `SchemaDeserializationError` may be raised for errors when calling `deserialize` on `AvroSerializer`.
+  - Depends on `azure-schemaregistry>=1.0.0,<2.0.0`.
```

Comment:
kitten nit, but I think it's over indented?
```suggestion
- Depends on `azure-schemaregistry>=1.0.0,<2.0.0`.
```

Reply:
ah yes

Reply:
good catch!